mlrun/projects/pipelines.py

```diff
@@ -696,6 +696,51 @@ class _RemoteRunner(_PipelineRunner):
 
     engine = "remote"
 
+    @staticmethod
+    def _prepare_load_and_run_function(
+        source,
+        project_name,
+        save,
+        workflow_name,
+        workflow_spec,
+        artifact_path,
+        workflow_handler,
+        namespace,
+    ):
+        # Creating the load project and workflow running function:
+        load_and_run_fn = mlrun.new_function(
+            name=f"workflow-runner-{workflow_name}",
+            project=project_name,
+            kind="job",
+            image=mlrun.mlconf.default_base_image,
+        )
+        runspec = mlrun.RunObject.from_dict(
+            {
+                "spec": {
+                    "parameters": {
+                        "url": source,
+                        "project_name": project_name,
+                        "save": save,
+                        "workflow_name": workflow_name or workflow_spec.name,
+                        "workflow_path": workflow_spec.path,
+                        "workflow_arguments": workflow_spec.args,
+                        "artifact_path": artifact_path,
+                        "workflow_handler": workflow_handler or workflow_spec.handler,
+                        "namespace": namespace,
+                        "ttl": workflow_spec.ttl,
+                        "engine": workflow_spec.engine,
+                        "local": workflow_spec.run_local,
+                    },
+                    "handler": "mlrun.projects.load_and_run",
+                },
+                "metadata": {"name": workflow_name},
+            }
+        )
```

Comment:
`from_dict` should be used for loading from db / files. If you have the data, create the spec, metadata and object in a pythonic way